test/framework/easyconfig.py

```diff
@@ -1199,10 +1199,17 @@ def test_dump_extra(self):
             "homepage = 'http://foo.com/'",
             'description = "foo description"',
             '',
-            "toolchain = {'version': 'dummy', 'name': 'dummy'}",
```

Comment:
actually, this should stay like this singe-line (by exception for the `toolchain*` keys)...